tests/Jackalope/Transport/DoctrineDBAL/CachedClientTest.php

```diff
@@ -45,7 +45,7 @@ public function testDefaultKeySanitizer()
         $this->cacheMock
             ->method('fetch')
             ->with(self::callback(function ($arg) use (&$first) {
-                self::assertEquals($first ? 'nodetypes:_a:0:{}' : 'node_types', $arg);
+                self::assertEquals($first ? 'nodetypes__a_0_{}' : 'node_types', $arg);
```

Comment:
If I look at the error message which @flogado posted here: https://github.com/sulu/sulu/issues/7394

I think we also run into issue with the `{}`?